control_msgs/action/JointGroupCommand.action

```diff
@@ -0,0 +1,34 @@
+# Used in time-stamping the goal.
+Header header
+
+# Name list of the joints. You don't need to specify all joints of the
+# robot. Joint names are case-sensitive.
+string[] joint_names
+
+# A command to the joints listed in joint_names.
+# The order must be identical.
+float64[] command
```

Comment:
Wold it make sense to make this command more explicit, e.g. `trajectory_msgs/JointTrajectoryPoint`. This would probably avoid future extensions and fields can be used explicitly (positions, velocities, accelerations).

Does this makes sense for you?

Reply:
Pros:
* Could allow for dual command types (e.g. position with a max velocity/acceleration)
* Explicit command is easier to interpret

Cons:
* Doesn't allow for non-traditional joint commands (only position, velocity, acceleration, effort) outside of overloading a variable
* Makes the coding in ros_controllers a bit more difficult (need to add logic that detects the command type and writes to the appropriate field)

I think the pros can outweigh the cons, so let me make the changes in both packages.